src/AppInstallerCLITests/InstallFlow.cpp

```diff
@@ -1204,3 +1204,68 @@ TEST_CASE("InstallFlow_InstallAcquiresLock", "[InstallFlow][workflow]")
     REQUIRE(installResultStr.find("/custom") != std::string::npos);
     REQUIRE(installResultStr.find("/silentwithprogress") != std::string::npos);
 }
+
+TEST_CASE("InstallFlow_InstallMultipleWithReboot", "[InstallFlow][workflow][MultiQuery][reboot]")

```

Comment:
Not a huge fan of a test that, if it "fails", reboots my machine...

Can we also have an override that replaces the reboot operation with a callback and you can just set a bool?  That would also allow for tests that actually "reboot".

Reply:
Changed to suggested and added tests that verifies if a reboot actually succeeds